build/mark-new-version.sh

```diff
@@ -92,6 +92,10 @@ fi
 
 echo "+++ Running ./versionize-docs"
 ${KUBE_ROOT}/build/versionize-docs.sh ${NEW_VERSION}
+
+echo "+++ Updating swagger"
+${KUBE_ROOT}/hack/update-generated-swagger-docs.sh
```

Comment:
Isn't this script already run once in versionize-docs.sh? https://github.com/kubernetes/kubernetes/blob/master/build/versionize-docs.sh#L74